ddspipe_participants/src/cpp/participant/rtps/SimpleParticipant.cpp

```
@@ -117,6 +118,30 @@ SimpleParticipant::reckon_participant_attributes_() const
             break;
     }
 
+    if (!simple_configuration->easy_mode_ip.empty())
+    {
+        // This option is incompatible with transport tag configurations,
+        // so check that transport is set to the default value (builtin)
+        if (simple_configuration->transport != core::types::TransportDescriptors::builtin)
+        {
+            EPROSIMA_LOG_WARNING(SIMPLEPARTICIPANT,
+                    "Ignoring IP value: Easy mode configuration is incompatible with transport tag configurations.");
+        }
+        else
+        {
+            // Check if the IP is a valid IPv4 address
+            if (!fastdds::rtps::IPLocator::isIPv4(simple_configuration->easy_mode_ip))
+            {
+                EPROSIMA_LOG_WARNING(SIMPLEPARTICIPANT,
+                        "Ignoring Easy Mode IP value. It must be a valid IPv4 address.");
+            }
+            else
+            {
+                params.easy_mode_ip = simple_configuration->easy_mode_ip;
+            }
+        }
+    }
```

I would move this logic to the `is_valid` method, and perhaps it would be better to fail instead of just printing a warning.

---

Changes applied on f0d98af5dfd1749899e249d0fc367a95b3c3b519